Xamarin.Forms.Platform.iOS/Renderers/IShellNavBarAppearanceTracker.cs

```diff
@@ -6,6 +6,7 @@ namespace Xamarin.Forms.Platform.iOS
 
 	public interface IShellNavBarAppearanceTracker : IDisposable
 	{
+		ShellAppearance CurrentAppearance { get; }
```

Comment:
It'd be nice to not have to modify this interface.  Can we just store the appearance locally to ShellSectionRenderer when `OnAppearanceChanged` is called?

Reply:
Changed, see the last comment added.